.eslintrc.js

```diff
@@ -283,19 +281,10 @@ const stylisticIssuesRules = {
 			tabWidth: 4
 		}
 	],
-	'max-lines': TRANSITION_WARNING,
```

Comment:
This seems mostly aspirational; it can easily be added back if it's ever addressed, but I'm not sure it always makes sense.